supervision/annotators/core.py

```diff
@@ -1975,7 +2006,7 @@ def __init__(
         border_color: Color = Color.BLACK,
         position: Position = Position.TOP_CENTER,
         color_lookup: ColorLookup = ColorLookup.CLASS,
-        border_thickness: int = None,
+        border_thickness: Optional[int] = None,
```

Comment:
Following your logic. shouldn't `border_thickness` be marked as `Optional[int]` in docs?

Reply:
It absolutely should. Thank you!